src/components/Calculator.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable react/self-closing-comp */
+/* eslint-disable react/no-unknown-property */
```

Comment:
- [ ] Kindly do not use the disable linter feature as linters are used to guide us to write industry standard code.